modules/gateway/gateway.go

```diff
@@ -271,6 +264,8 @@ func New(addr string, bootstrap bool, persistDir string) (*Gateway, error) {
 			if err != nil && err != errNodeExists {
 				g.log.Printf("WARN: failed to add the bootstrap node '%v': %v", addr, err)
 			}
+			// All bootstrap nodes should be prioritized.
+			g.prioritizeNode(addr)
```

Comment:
I don't think you should prioritize the bootstrap node, they are obvious targets for attack.